source/includes/oai-pmh/_root.md

```diff
@@ -143,28 +186,58 @@ correct community identifier.
 [See example](https://zenodo.org/oai2d?verb=ListRecords&
 metadataPrefix=oai_datacite&set=user-cfa)
 
-If you need selective harvesting and your use case is not supported by above
-sets, please [contact us](http://about.zenodo.org/contact/) and we may possible
-create a specific set for you.
 
-## Update schedule
+```python
+# Fetch a couple of records from the OAI Set of the "cfa" community
+>>> records = sickle.ListRecords(metadataPrefix='oai_dc', set='user-cfa')
+>>> record = records.next()
 
-Most updates are available immediately, some few updates are only reflected in the OAI sets once an hour.
+# To inspect on what sets a record is in
+>>> record.header.setSpecs
+<...>
```

Comment:
```suggestion
[<Set user-cfa>, <Set software>]  # not sure if this is what's displayed, so check the output
```
I would add explicit examples here, otherwise there's nothing to show.

Reply:
Makes sense!